src/solve.jl

```diff
@@ -190,33 +210,14 @@ function solve!(system::InitialValueProblem{<:HybridSystem,
                     end
                 end
             else
-                for (i, reach_set) in enumerate(reach_tube.Xk)
+                 for (i, reach_set) in enumerate(Rsets[length(Rsets) - count_Rsets + 1 : end])
```

Comment:
I do not understand this change.

Reply:
See [this line](https://github.com/JuliaReach/Reachability.jl/blob/master/src/ReachSets/DiscretePost/LazyDiscretePost.jl#L103). `reach_tube` is a result of continuous post, while `Rsets` contains sets intersected with invariant. However, since Rsets contains all the sets, we need to filter only those we get during last `c_post`